astroid/manager.py

```diff
@@ -78,12 +78,12 @@ def __init__(self) -> None:
         self._transform = AstroidManager.brain["_transform"]
 
     @property
-    def register_transform(self):
+    def register_transform(self) -> Callable[[Any, Any, Any], None]:
```

Comment:
Let's not do this. We're actually losing information here as the current return type seen in IDEs will include the default `None` value for the third argument.